src/components/Calculator/Digit.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+// 버튼을 눌르면 Total Component가 업데이트
+type DigitProps = {
+    handleClick: React.MouseEventHandler<HTMLButtonElement>;
+};
+
+export const Digit = ({ handleClick }: DigitProps) => {
+    const digitButtons=[0,1,2,3,4,5,6,7,8,9].reverse()
+    const renderButton=digitButtons.map((number)=><button value={number} key={number} onClick={handleClick}>{number}</button>)
+
+    return (
+        <div className="digits flex">
+            {renderButton}
+        </div>
+    );
```

Comment:
```suggestion

    return (
        <div className="digits flex">
            {digits.map((digit) =>
                <button value={digit} key={digit} onClick={handleClick}>{digit}</button>)
            }
        </div>
    );
```
(마이너) 저는 렌더를 별도로 할당하지 않고 바로 return 내부에서 적어주는걸 더 선호해요.
이러한 로직들이 많아지면 return문 내에서 코드 흐름을 따라가기위해 계속 아래위를 왔다갔다 해야하는 문제점이 있을 것 같기도해서요.

추가적으로 renderButton이라기엔 조금 네이밍이 어색한 것 같아요.
render가 렌더링을 유발하는 함수라고 유추하기 쉬운 네이밍인데 실제로는 렌더링에 포함될 Element의 배열이 할당된 상태라서요 🤔